test/files/run/t6989.check

```diff
@@ -107,12 +107,14 @@ isProtected = false
 isPublic = true
 privateWithin = <none>
 ============
+#partest !java20+
```

Comment:
I wondered if this was legit or might be some kind of regression, but on 2.13.x the same change was already made in d94357238822e094f221c9e09056dc3607ba256c